receiver/webhookeventreceiver/config.go

```diff
@@ -72,6 +73,14 @@ func (cfg *Config) Validate() error {
 		}
 	}
 
+	// Set default MaxRequestBodySize if not configured
+	if cfg.MaxRequestBodySize == 0 {
+		cfg.MaxRequestBodySize = int64(bufio.MaxScanTokenSize)
+	} else if cfg.MaxRequestBodySize < int64(bufio.MaxScanTokenSize) {
+		// Enforce minimum of 64KB (bufio.MaxScanTokenSize)
+		cfg.MaxRequestBodySize = int64(bufio.MaxScanTokenSize)
+	}
```

Comment:
Why does a minimum value need to be enforced? I don't think we need this `else`.

Reply:
Removed the minimum. 🫡